lib/claim/token-holder-revenue-fund-contracts-ensemble.js

```diff
@@ -18,33 +18,41 @@ const minBigNumber = (n1, n2) => n1.lt(n2) ? n1 : n2;
  * @param {Currency} currency - The currency
  */
 async function span(currency) {
-    const ensemble = _ensemble.get(this);
-    let globalAccrualOffset = _firstAccrualOffset.get(this);
+    const configurations = _configurations.get(this);
+    let accrualOffset = _firstAccrualOffset.get(this);
+    let blockOffset = 0;
     const currencyKey = currencyToKey(currency);
 
-    for (const configuration of ensemble) {
+    for (const configuration of configurations) {
         const closedAccrualsCount = (await configuration.contract.closedAccrualsCount(
             currency.ct.toString(), currency.id
         )).toNumber();
 
-        const span = {};
-        span.firstAccrual = globalAccrualOffset || 0;
-        span.lastAccrual = span.firstAccrual + closedAccrualsCount - 1;
-        span.firstBlock =
-            (await configuration.contract.closedAccrualsByCurrency(
-                currency.ct.toString(), currency.id, span.firstAccrual
-            )).startBlock.toNumber();
-        span.lastBlock =
-            (await configuration.contract.closedAccrualsByCurrency(
-                currency.ct.toString(), currency.id, span.lastAccrual
-            )).endBlock.toNumber();
+        if (0 === closedAccrualsCount)
+            continue;
+
+        const firstBlock = (await configuration.contract.closedAccrualsByCurrency(
+            currency.ct.toString(), currency.id, 0
+        )).startBlock.toNumber();
+        const lastBlock = (await configuration.contract.closedAccrualsByCurrency(
```

Comment:
Pedantic comment: Would it make sense to insert a blank line before `const lastBlock = ...`?

Reply:
👍